src/main/java/com/nasrpi/home/HomeController.java

```diff
@@ -50,14 +63,26 @@ public boolean createDirectory(@RequestBody final String directoryPath) {
 		return homeRepository.createDirectory(directoryPath);
 	}
 
-	@RequestMapping(value = "/moveFolder", method = RequestMethod.POST, produces = "application/json")
+	@RequestMapping(value = "/moveFolder", method = RequestMethod.POST, produces = KeyConstants.APPLICATION_JSON)
 	public boolean moveFolder(@RequestBody final MoveModel moveModel) {
 		return homeRepository.moveFolder(moveModel.getSource(), moveModel.getDestination());
 	}
-	
-	@RequestMapping(value = "/moveFile", method = RequestMethod.POST, produces = "application/json")
+
+	@RequestMapping(value = "/moveFile", method = RequestMethod.POST, produces = KeyConstants.APPLICATION_JSON)
 	public boolean moveFile(@RequestBody final MoveModel moveModel) {
 		return homeRepository.moveFile(moveModel.getSource(), moveModel.getDestination());
 	}
 
+	@PostMapping("/uploadFile")
```

Comment:
change this to request mapping

Reply:
okay. is it due to any functional limitations of @PostMapping or just a good programming practice to use @RequestMapping?

Reply:
done